bool activation_state
---
bool success
string message
